lib/index.ts

```diff
@@ -33,10 +33,7 @@ const MAX_FILTER_DEPTH = 10;
  * @param processList The list of processes
  * @param maxDepth The maximum depth to search
  */
-export function buildProcessTree(rootPid: number, processList: IProcessInfo[] | undefined, maxDepth: number): IProcessTreeNode | undefined {
-  if (!processList) {
-    return undefined;
-  }
+export function buildProcessTree(rootPid: number, processList: IProcessInfo[], maxDepth: number = MAX_FILTER_DEPTH): IProcessTreeNode | undefined {
```

Comment:
Should we just remove the `maxDepth` arg as I don't think it's used anymore and use the constant directly?

Reply:
The initial starting value is overridden in the recursive calls, so the parameter itself can't be removed.

An alternative I considered vs using a default argument here was bind it in the initial transform argument:

```ts
getRawProcessList(
  rootPid,
  processTreeRequestQueue,
  callback,
  (pid, list) => buildProcessTree(pid, list, MAX_DEPTH),
  flags,
);
```

But that seemed like a noisier change.

Reply:
Ah I missed that, let's keep it